protocol.go

```diff
@@ -108,6 +118,22 @@ type GitSignResponse struct {
 	Error     *string `json:"error,omitempty"`
 }
 
+type HostsRequest struct{}
+
+type UserAndHost struct {
+	User string `json:"user,omitempty"`
```

Comment:
omitempty should only be used on pointer members

Reply:
right, 👍